datahub-graphql-core/src/main/resources/gms.graphql

```diff
@@ -1807,3 +1820,159 @@ type Deprecation {
     """
     actor: String
 }
+
+type DataFlow implements Entity {
+    """
+    The unique flow URN
+    """
+    urn: String!
+
+    """
+    GMS Entity Type
+    """
+    type: EntityType!
+
+    """
+    Workflow orchestrator ex: Azkaban, Airflow
+    """
+    orchestrator: String!
+
+    """
+    Id of the flow
+    """
+    flowId: String!
+
+    """
+    Cluster of the flow
+    """
+    cluster: String!
+
+    """
+    Information about a Data processing flow
+    """
+    info: DataFlowInfo
+
+    """
+    Ownership metadata of the flow
+    """
+    ownership: Ownership
+}
+
+type DataFlowInfo {
+    """
+    Name of the flow
+    """
+    name: String!
+
+    """
+    Description of the flow
+    """
+    description: String
+
+    """
+    Optional project/namespace associated with the flow
+    """
+    project: String
+}
+
+
+type DataJob implements Entity {
+    """
+    The unique job URN
+    """
+    urn: String!
+
+    """
+    GMS Entity Type
+    """
+    type: EntityType!
+
+    """
+    The associated data flow
+    """
+    dataFlow: DataFlow!
```

Comment:
Modeling question: What if a DataJob is "orphaned". That is, there is no parent DataFlow? (It is run in an ad-hoc manner). 

Would someone want to be able to model this? If so, how would we advise they do so?

Reply:
Well we have the `DataProcessor` which has the the inputs/outputs of the DataJob and the orchestrator of the DataFlow. I think that would be the best option in this case.

Reply:
Not familiar with DataProcess - taking a look